src/js/settings.js

```diff
@@ -0,0 +1,61 @@
+import { setting } from './translation.js'
+
+const settings = document.querySelector('.settings');
+const settingsBtn = document.querySelector('.icon_settings');
+const elementsList = document.querySelectorAll('.block input');
+const elementsListText = document.querySelectorAll('.block label');
+const textCahngeLang = document.querySelector('.title__language');
+const textChangePhoto = document.querySelector('.title__photo');
+const ruLang = document.querySelector('option[value="ru"]');
+const enLang = document.querySelector('option[value="en"]');
+
+let isOpen = false;
+
+export const openCloseSettings = () => {
+  settingsBtn.addEventListener('click', () => {
+    if (isOpen === false) {
+      isOpen = true;
+      settings.style.transform = `translateX(0)`;
+    } else if (isOpen === true) {
+      isOpen = false;
+      settings.style.transform = `translateX(-120%)`;
+    }
+  })
+};
+
+export const showHideElements = () => {
```

Comment:
the name of the function does not reflect what it does. smth like `setUpEventListenersForSettings` might be better. think about your own name